lib/generators/scenic/view/view_generator.rb

```diff
@@ -85,6 +85,10 @@ def previous_definition
         Scenic::Definition.new(plural_file_name, previous_version)
       end
 
+      def plural_file_name
+        @plural_file_name ||= file_name.pluralize.gsub('.', '_')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.